apps/web/src/features/spaces/components/SpaceAddressBook/index.tsx

```diff
@@ -17,7 +19,52 @@ const SpaceAddressBook = () => {
   }
 
   // TODO: Get data from CGW
-  const entries = []
+  const entries = [
+    {
+      address: '0xF94c38db9992cfE106C6502bfB6efa58519f7570',
+      name: 'John Doe',
+      networks: [
+        {
+          chainId: '11155111',
```

Comment:
We should discuss with @hectorgomezv what data to store on the CGW. For example would it be sufficient to only store the chain id and a name or do we need more information like the chain name (for convenvience).

Reply:
For the scope of this PR lets leave it as is though since we still need to decide on a prefered structure